solr/solrj/src/java/org/apache/solr/client/solrj/impl/ConcurrentUpdateHttp2SolrClient.java

```diff
@@ -256,7 +256,7 @@ void sendUpdateStream() throws Exception {
             }
 
             Response response =
-                responseListener.get(client.getIdleTimeout(), TimeUnit.MILLISECONDS);
+                responseListener.get(client.getRequestTimeoutMillis(), TimeUnit.MILLISECONDS);
```

Comment:
It was weird to be using the idle (aka socket) timeout in a situation where we're waiting for the request.